satpy/readers/ahi_hsd.py

```diff
@@ -499,9 +527,20 @@ def _mask_space(self, data):
         """Mask space pixels."""
         return data.where(get_geostationary_mask(self.area))
 
+    def _get_rad_corr_factors(self):
+        slope = self.radiance_correction[self.band_name]['slo']
+        offset = self.radiance_correction[self.band_name]['off']
+
+        return slope, offset
+
+    def _apply_rad_correction(self, data, slo, off):
+        """Apply GSICS-like correction factors to radiance data."""
+        data = (data - off) / slo
+
+        return data
```

Comment:
This method is duplicated (in ami), and is static. Please make it a function in a shared module.

Reply:
Done.